package.json

```diff
@@ -30,33 +30,36 @@
     "build:dev": "npm run remove-rcue; REACT_APP_RCUE=false npm run build:kiali",
-    "build:kiali": "npm run lint:prod && npm run copy-fonts && npm run copy-img && npm run build-css && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) react-scripts-ts build",
+    "build:kiali": "npm run lint && npm run copy-fonts && npm run copy-img && npm run build-css && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) react-scripts build",
     "build:prod": "npm run restore-rcue; REACT_APP_RCUE=true npm run build:kiali",
     "copy-fonts": "cp -r node_modules/patternfly/dist/fonts src/",
     "copy-img": "cp -r node_modules/patternfly/dist/img src/",
-    "lint": "tslint --project ./tsconfig.json",
-    "lint:prod": "tslint --project ./tsconfig.json --config tslint.prod.json",
-    "lint:precommit": "if git diff --name-only HEAD | grep -E '\\.tsx?$'; then npm run lint:prod; else true; fi",
-    "lintfix": "tslint --fix --project ./tsconfig.json",
+    "lint": "eslint --ext js,ts,tsx src",
+    "lint:precommit": "if git diff --name-only HEAD | grep -E '\\.tsx?$'; then npm run lint; else true; fi",
+    "lintfix": "eslint --ext js,ts,tsx --fix src",
     "remove-rcue": "npm run backup-rcue ; npm run backup-rcue-additions",
     "restore-rcue": "mv node_modules/patternfly/dist/css/rcue.css.copy node_modules/patternfly/dist/css/rcue.css && mv node_modules/patternfly/dist/css/rcue-additions.css.copy node_modules/patternfly/dist/css/rcue-additions.css",
     "start": "if [ \"${KIALI_ENV}\" = \"production\" ]; then npm run start:prod; else npm run start:dev; fi",
     "start:dev": "npm run remove-rcue; REACT_APP_RCUE=false npm run start:kiali",
-    "start:kiali": "npm run copy-fonts && npm run copy-img && npm run build-css && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) react-scripts-ts start",
+    "start:kiali": "npm run copy-fonts && npm run copy-img && npm run build-css && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) react-scripts start",
     "start:prod": "npm run restore-rcue; REACT_APP_RCUE=true npm run start:kiali",
-    "test": "TEST_RUNNER=1 react-scripts-ts test --env=jsdom __tests__",
+    "test": "TEST_RUNNER=1 react-scripts test --env=jsdom __tests__",
     "snyk": "snyk test",
     "analyze": "source-map-explorer build/static/js/main.*",
     "prettier": "prettier --write \"{src/**/*.{js,jsx,ts,tsx,json,yml,css,scss},travis.yml,*.json}\"",
-    "snyk-protect": "snyk protect"
+    "snyk-protect": "snyk protect",
+    "postinstall": "sed -i -E 's#\\[.*eslint-config-react-app.*\\]#['\\'$PWD/.eslintrc\\'']#g' node_modules/react-scripts/config/webpack*.js"
```

Comment:
I don't totally understand this little ditty...

Reply:
It changes from the default path for `react-scripts`, to stop using the default `eslint` configuration inside `node_modules` to use `.eslintrc` on the project home every time you install the dependencies.

Reply:
@mtho11 did you test installing the dependencies on os x? I just remembered `sed` has different usage on mac, can you check if it works correctly?

If it doesn't fail and you can run `yarn lint` without any errors then everything is ok :)

Reply:
Because JSON can't really be commented, I would recommend that this "black magic" explanation be recorded in the final commit comment.  In general, I think we lack explanations for some of the tricky code we leave behind.  There is nothing wrong with helping a future dev with inline or in-commit comments.  Info left in a PR is not as easily traceable.

Reply:
@jshaughn check this out: https://github.com/kiali/kiali-ui/pull/1154/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R35

But I can add it to the commit message with a more detailed explanation if you think that's not enough.

Reply:
@cfcosta works fine on OS/X.